test/es-module/test-esm-throw-undefined.mjs

```diff
@@ -0,0 +1,15 @@
+// Flags: --experimental-modules
+/* eslint-disable node-core/required-modules */
+import common from '../common/index.js';
+import assert from 'assert';
+
```

Comment:
nit: add a `common.crashOnUnhandledRejection();` line here, which does away with the `.then(common.mustCall())` later.

Reply:
Thank you. As this is the first time I make a pull request I'm not very familiar with the these functions. You can add it later :)

Reply:
@TimothyGu I added.